servicetalk-test-resources/src/main/resources/junit-platform.properties

```diff
@@ -0,0 +1,8 @@
+#
+# See https://junit.org/junit5/docs/snapshot/user-guide/index.html#writing-tests-parallel-execution
+#
+junit.jupiter.execution.parallel.enabled = true
+junit.jupiter.execution.parallel.mode.default = same_thread
+junit.jupiter.execution.parallel.mode.classes.default = same_thread
+junit.jupiter.execution.parallel.config.strategy = dynamic
+junit.jupiter.execution.parallel.config.dynamic.factor = 1.5
```

Comment:
How about starting with 1 as we discussed before: https://github.com/apple/servicetalk/pull/1588#discussion_r643575718?

Reply:
Sure.